run-qa-checks

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+openwisp-qa-check \
+    --migration-path ./openwisp_firmware_upgrader/migrations \
+    --migration-module firmware_upgrader
+
+# JShint checks
+jshint openwisp_firmware_upgrader/static/firmware-upgrader/js/*.js
+
+echo ''
+echo 'Running checks for SAMPLE_APP'
+SAMPLE_APP=1 openwisp-qa-check \
+    --skip-isort \
+    --skip-flake8 \
+    --skip-checkmigrations \
+    --skip-checkendline \
+    --skip-checkcommit \
+    --skip-black \
+    --migration-path ./tests/openwisp2/sample_firmware_upgrader/migrations/ \
+    --migration-module sample_firmware_upgrader
```

Comment:
did you just copy paste the script from firmware-upgrader module without changing it? I see firmware-upgrader in every command, you should update to match the directories of this repository.

Reply:
Yes actually but would update it, sorry was not aware, and thank you.